package.json

```diff
@@ -63,9 +65,9 @@
     "vite-plugin-environment": "^1.1.3",
     "vite-plugin-vuetify": "^1.0.2",
     "vitest": "^0.33.0",
+    "vitest-axe": "^0.1.0",
```

Comment:
What's this for?

Reply:
This is to import vitest-axe/extend-expect in test setup file to make sure tests work with Vitest

Ref: https://github.com/chaance/vitest-axe